### DISCUSSION TITLE
How can I build Sandboxie Plus using Docker?

### DISCUSSION THREAD

**Discussion by Aholicknight** [sandboxie-plus/Sandboxie#3480] [Q&A Feedback] [answered]
Hello. I would love to help with SandboxiePlus but stuck on how to build. Is there any documentation available? Is it possible to build with Docker? Of course, I can build using GitHub Actions, but that takes significantly longer.

Thank you!

**Comment by isaak654** [comment, marked as answer]
The main workflow provided as a GitHub Action has been optimized since your request and, at the time of writing, takes 6 to 9 minutes.

About Docker, there is nothing about it so far. If you want to check a similar issue, see #3717
